pkg/controller/certificaterequests/selfsigned/selfsigned_test.go

```diff
@@ -542,6 +614,7 @@ type testT struct {
 
 func runTest(t *testing.T, test testT) {
 	test.builder.T = t
+	//test.builder.Recorder = new(testpkg.FakeRecorder)
```

Comment:
Remove

Reply:
Still needs to be removed :sweat_smile:

Reply:
I'm an expert at finding ways to shoot myself in the foot with git :+1: